_posts/2022-08-08-understanding-meta-keyword-argument.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: Understanding Dask’s meta keyword argument
+author: Pavithra Eswaramoorthy
+tags: [dataframe]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+If you have worked with Dask DataFrames or Dask Arrays, you have probably come across the `meta` keyword argument and seen one or more of the following warning/error messages:
```

Comment:
Could you add a small code example showing `meta` being used here?

Reply:
Good idea, done!